<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Показать запись</string>
  </property>
  <widget class="QLabel" name="lName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Названние записи</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="tText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
